.devcontainer/postCreate.sh

```diff
@@ -2,7 +2,7 @@
 
 pip install --upgrade pip
 
-pip install -r code/requirements.txt -r code/backend/requirements.txt -r code/backend/batch/requirements.txt
+pip install -r code/requirements.txt -r code/backend/requirements.txt 
```

Comment:
Did you mean to delete `-r code/backend/batch/requirements.txt`? Isn't that needed?

Reply:
good catch